src/components/manage/Widgets/ColorPickerWidget.stories.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ColorPickerWidget from './ColorPickerWidget';
+import WidgetStory from './story';
+
+export const Default = WidgetStory.bind({
+  widget: ColorPickerWidget,
+});
+
+Default.args = {
+  id: 'localhost:3000/something',
```

Comment:
@danalvrz The `id` used by this widget is a field id, not the full path for the page whose data includes that field. I'd suggest using `favoriteColor` as the example id here, and "Favorite Color" as the title.

(We should also use a more realistic field id and title in the stories for the other widgets.)